tests/decoder_tests.rs

```diff
@@ -356,6 +356,97 @@ fn gainmap_oriented() {
     assert_eq!(decoder.gainmap().image.imir_axis, None);
 }
 
+// From avifgainmaptest.cc
+#[test]
+fn decode_unsupported_version() {
+    // The two test files should produce the same results:
+    // One has an unsupported 'version' field, the other an unsupported
+    // 'minimum_version' field, but the behavior of these two fiels is the same.
+    for image in [
```

Comment:
this loop is not necessary, you can parameter'ize this function to take a file name so that the tests will run on each file.

see the `progressive()` function in line 185 above for an example.